pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -2531,6 +2531,12 @@ func (dm *DockerManager) GetNetNS(containerID kubecontainer.ContainerID) (string
 		glog.Errorf("Error inspecting container: '%v'", err)
 		return "", err
 	}
+	if inspectResult.State.Pid == 0 {
```

Comment:
please log something like "cannot find net ns of TERMNINATED container ID in pod NAME"